mmv1/templates/terraform/examples/redis_cluster_cmek.tf.tmpl

```diff
@@ -0,0 +1,49 @@
+resource "google_redis_cluster" "{{$.PrimaryResourceId}}" {
+  name           = "{{index $.Vars "cluster_name"}}"
+  shard_count    = 3
+  psc_configs {
+    network = google_compute_network.consumer_net.id
+  }
+  kms_key = "{{index $.Vars "kms_key_name"}}"
+  region = "us-central1"
+  deletion_protection_enabled = {{index $.Vars "deletion_protection_enabled"}}
+  depends_on = [
+    google_network_connectivity_service_connection_policy.default,
+    google_kms_crypto_key_iam_binding.kms_key_iam_decrypter
+  ]
+}
+
+resource "google_kms_crypto_key_iam_binding" "kms_key_iam_decrypter" {
+  crypto_key_id = "{{index $.Vars "kms_key_name"}}"
+  role          = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  members       = [
+    "serviceAccount:service-${data.google_project.project.number}@cloud-redis.iam.gserviceaccount.com",
+  ]
+}
```

Comment:
this should be done by bootstrapping this permission at the project level with `bootstrap_iam`: https://googlecloudplatform.github.io/magic-modules/reference/resource/#examples

```suggestion
```